src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/BitcoinActivationRule.cs

```diff
@@ -23,31 +23,10 @@ public override Task RunAsync(RuleContext context)
         {
             Guard.NotNull(context.ConsensusTip, nameof(context.ConsensusTip));
```

Comment:
The exceptions above are no longer thrown here. Also the context parameter should be inherited